src/components/inspectors/SignalSelect.vue

```diff
@@ -347,3 +353,9 @@ export default {
     padding: 0px
   }
 </style>
+
+<style>
+  .invalid .multiselect__tags {
+    border-color:#f04124!important
```

Comment:
This border color is different from the border color of other invalid elements in the inspector panel:
<img width="588" alt="Screen Shot 2020-04-20 at 10 23 58 AM" src="https://user-images.githubusercontent.com/7561061/79762725-1e948300-82f1-11ea-96e6-8c723c5d035b.png">

It looks like the other inputs have an invalid border color of `#dc3545`.

The invalid style that applies the above red color comes from vue-form-elements:
https://github.com/ProcessMaker/vue-form-elements/blob/711586cc154f1312f4cd22d8d226b216ae6d28d1/src/components/FormHtmlViewer.vue#L27-L30